graphql-ballerina/build.gradle

```diff
@@ -159,23 +158,29 @@ test {
 
 task ballerinaBuild {
     inputs.dir file(project.projectDir)
-    dependsOn(":graphql-native:build")
-    dependsOn(unpackJballerinaTools)
-    dependsOn(unpackStdLibs)
-    dependsOn(copyStdlibs)
     dependsOn(updateTomlFile)
-    dependsOn(test)
+    dependsOn(initializeVariables)
     finalizedBy(revertTomlFile)
 
+    def testParams = "--skip-tests"
+    gradle.taskGraph.whenReady { graph ->
+        if (graph.hasTask(':graphql-ballerina:build')) {
+            ballerinaTest.enabled = false
+        }
+        if (graph.hasTask(':graphql-ballerina:test')) {
+            testParams = "--code-coverage"
+        }
+    }
+
     doLast {
         // Build and populate caches
         exec {
             workingDir project.projectDir
             environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
             if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/bal.bat build --skip-tests -c"
+                commandLine 'cmd', '/c', "$balJavaDebugParam $distributionBinPath/bal.bat build $testParams ${debugParams} && exit %%ERRORLEVEL%%"
```

Comment:
We can keep the `-c` flag. I think it's there to avoid creating the executable jar.

Reply:
If we are merging this after the lang repo PR merged, then we can keep the `--include=*` flag added to the command